docs/rabbitmq.config.example

```diff
@@ -246,6 +246,10 @@
    %%
    %% {memory_monitor_interval, 2500},
 
+   %% The total memory available can be calculated from the OS resources
+   %% - default option - or provided as a configuration parameter:
+   %% {total_memory_available_override_value, "5000MB"},
```

Comment:
⚠️ nitpick ahead!

Since this value is parsed via `rabbit_resource_monitor_misc:parse_information_unit`, `5000MB` will be interpreted as the powers-of-10 value, rather than powers-of-2. The user may not expect this behavior, I'm not sure.

Related to rabbitmq/rabbitmq-server#1233 and rabbitmq/rabbitmq-server#1235